simplified-viewer-audiobook/src/main/java/org/librarysimplified/viewer/audiobook/AudioBookPlayerActivity.kt

```diff
@@ -848,7 +848,7 @@ class AudioBookPlayerActivity :
      */
 
     this.uiThread.runOnUIThread {
-      val fragment = PlayerSleepTimerFragment.newInstance()
+      val fragment = PlayerSleepTimerFragment.newInstance(PlayerFragmentParameters())
```

Comment:
This shouldn't be here because when the audiobook submodule versions are correctly implemented and imported, we'll have an error